tests/test_stubs.py

```diff
@@ -1,34 +1,60 @@
-import os.path
+from pathlib import Path
 import pytest
 from mypy import api
 
 
-TESTS_DIR = os.path.dirname(__file__)
+TESTS_DIR = Path(__file__).parent
 
 
 def gen_tests():
-    for filename in os.listdir(TESTS_DIR):
-        if filename.endswith('.py') and not filename.startswith('test_'):
+    '''List of all tests files included in the directory tests'''
+    for filename in TESTS_DIR.glob('*.py'):
+        if not str(filename.parts[-1]).startswith('test_'):
             yield filename
```

Comment:
`.parts` is already strings, but the last one is `.name` anyways.

```suggestion
    for path in TESTS_DIR.glob('*.py'):
        if not path.name.startswith('test_'):
            yield path
```